dotCMS/src/test/java/com/dotmarketing/portlets/contentlet/model/ResourceLinkTest.java

```diff
@@ -194,25 +241,36 @@ public void test_vtl_ResourceLink_WithAdminUser_Expect_Downloadable_No_Port_Numb
         final ResourceLinkBuilder resourceLinkBuilder = getResourceLinkBuilder(hostName, path, mimeType, htmlFileName);
         final ResourceLink link = resourceLinkBuilder.build(request, adminUser, contentlet);
         assertFalse(link.isDownloadRestricted());
-        assertEquals("http://demo.dotcms.com/application/comments/angular/widget-code.vtl?language_id=2",link.getResourceLinkAsString());
+        assertEquals("http://demo.dotcms.com/application/comments/angular/"+htmlFileName+"?language_id=2",link.getResourceLinkAsString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "?language_id=2" appears 4 times in this file; the first occurrence is on line 229](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5372524)